### PR TITLE
add recipe 'Coco Loco'

### PR DESCRIPTION
**Pull request by lucasmartinelle**

### CODE REVIEW

File: src/recipes/coco-loco.json
Comment (alfg) [inline src/recipes/coco-loco.json:40]:
```suggestion
        "alcohol",

```

File: src/recipes/coco-loco.json
Comment (alfg) [inline src/recipes/coco-loco.json:3]:
```suggestion
    "description": "A rum and coconut cocktail from the West Indies.",

```

**Review by alfg — CHANGES_REQUESTED** [review]
I think this was branched from your last PR. So I can't merge until the last one is merged/approved. I also added a couple suggestions. Thanks.

**Comment by lucasmartinelle** [comment]
done :)

**Comment by alfg** [comment]
Hi, I think you'll need to sync up this branch with latest from master in order for the updated changeset to show. And to resolve conflict.

**Pull request closed without merging**
